ui/pages/v2/ApplicationList/ApplicationList.tsx

```diff
@@ -1,52 +1,97 @@
 import * as React from "react";
 import styled from "styled-components";
-import Button from "../../../components/Button";
 import DataTable from "../../../components/DataTable";
+import KubeStatusIndicator from "../../../components/KubeStatusIndicator";
 import Link from "../../../components/Link";
 import Page from "../../../components/Page";
-import { useListApplications } from "../../../hooks/apps";
-import { App } from "../../../lib/api/app/apps.pb";
-import { V2Routes } from "../../../lib/types";
+import { Automation, useListAutomations } from "../../../hooks/kustomizations";
+import { AutomationType, V2Routes } from "../../../lib/types";
 import { formatURL } from "../../../lib/utils";
-import AppOnboarding from "./AppOnboarding";
 
 type Props = {
   className?: string;
 };
 
 function ApplicationList({ className }: Props) {
-  const { data = { apps: [] }, isLoading, error } = useListApplications();
+  const { data, isLoading, error } = useListAutomations();
 
   return (
     <Page
-      title="Applications"
-      error={error?.code === 13 ? null : error}
+      title="Releases"
+      error={error}
       loading={!error && isLoading}
       className={className}
-      actions={
-        <Link to={formatURL(V2Routes.NewApp)}>
-          <Button>Add Application</Button>
-        </Link>
-      }
     >
-      {error || data?.apps.length === 0 ? (
-        <AppOnboarding />
-      ) : (
-        <DataTable
-          sortFields={["name"]}
-          fields={[
-            {
-              label: "Name",
-              value: ({ name }: App) => (
-                <Link key={name} to={formatURL(V2Routes.Application, { name })}>
-                  {name}
-                </Link>
-              ),
+      <DataTable
+        sortFields={[
+          "name",
+          "namespace",
+          "type",
+          "cluster",
+          "status",
+          "last synced at",
```

Comment:
I'm curious if my sort is handling "last synced at" correctly...

Reply:
Probs not; we will need to add the concept of data types.

Reply:
https://github.com/weaveworks/weave-gitops/issues/1467